hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/ConsumerMessageSender.java

```diff
@@ -101,6 +104,23 @@ private void handleFailedSending(Message message, MessageSendingResult result) {
             rateLimiter.registerSuccessfulSending();
         }
         errorHandler.handleFailed(message, subscription, result);
+        registerFailureMetrics(result);
+    }
+
+    private void registerSuccessMetrics(MessageSendingResult result) {
+        hermesMetrics.registerConsumerHttpAnswer(subscription, result.getStatusCode());
+    }
+
+    private void registerFailureMetrics(MessageSendingResult result) {
```

Comment:
Right now metrics in consumer are invoked mainly in Default{Succes,Error}Handler. I would also moved this metrics to mentioned handlers - it will be better to keep them in one place.